controls/src/main/java/com/aol/mobile/sdk/controls/ContentControls.java

```diff
@@ -100,6 +102,8 @@
         public boolean isOnLiveEdge;
         public boolean isCastButtonVisible;
         public boolean isCasting;
+        @NonNull
+        public final Set<Float> midrolls = new HashSet<>();
```

Comment:
Why `Float`? We are using `double` as progress for seeker. Also name `midrolls` confuses. My proposal is `adCues`